cmd/spoc/main.go

```diff
@@ -94,6 +97,62 @@ func main() {
 				},
 			},
 		},
+		&cli.Command{
+			Name:      "push",
+			Aliases:   []string{"p"},
+			Usage:     "push a profile to a container registry",
+			Action:    push,
+			ArgsUsage: "FILE",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:        pusher.FlagProfile,
+					Aliases:     []string{"f"},
+					Usage:       "the profile to be used",
+					DefaultText: pusher.DefaultInputFile,
+					TakesFile:   true,
+				},
+				&cli.StringFlag{
+					Name:    pusher.FlagUsername,
+					Aliases: []string{"u"},
+					EnvVars: []string{"USERNAME"},
+					Usage:   "the username for registry authentication",
+				},
+				&cli.StringFlag{
+					Name:    pusher.FlagPassword,
+					Aliases: []string{"p"},
```

Comment:
Is the password going to be provided in plain text in the argument? Maybe we should discourage this and just allow the environment variable or config file to avoid a situation when the password might be leaked in CI/CD.